erpnext/stock/doctype/batch/batch.py

```diff
@@ -414,8 +470,13 @@ def get_active_batch(item_code):
 				"certificate_of_analysis": "/files/coa.pdf"
 			}
 	"""
-
 	fields = ["name", "item", "item_name", "stock_uom", "thc", "cbd", "certificate_of_analysis", "batch_qty"]
 	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1}, fields=fields)
```

Comment:
```suggestion
	active_batch = frappe.get_all("Batch", filters={"item": item_code, "display_on_website": 1}, fields=fields, limit=1)
```

Reply:
If you want to only the first value, use `get_value` instead of `get_all`